OneSchedule.Domain/StateMachine/FindStateContext.cs

```diff
@@ -0,0 +1,72 @@
+﻿using OneSchedule.Attributes;
+using OneSchedule.Data.Abstractions;
+using OneSchedule.Domain.Abstractions.StateMachine;
+using OneSchedule.Domain.Models;
+using OneSchedule.Entities;
+using OneSchedule.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OneSchedule.Domain.StateMachine
+{
+    [StateContextName("Find")]
+    public class FindStateContext : IStateContext
+    {
+        private const string NextState = "AskEventList";
+        private IState _state;
+        private readonly IRepository<ContextEntity> _contextRepository;
```

Comment:
Again, there is a duplicated logic in Create state context and probably in other states too. Move it to base class